src/components/carousel/NoticesCarousel.jsx

```diff
@@ -130,7 +133,10 @@ export const NoticesCarousel = ({ content, title }) => {
   const slides = records.map((element) => (
     <>
       <Card key={element.codiceBando} className={classes.heroCards} spacing noWrapper>
-        <ExternalLink>
+        <ExternalLink
```

Comment:
I'd use internal link, I would not open a new tab here

Reply:
I changed the link type